File: Score of a String.cpp
class Solution {
public:
    int scoreOfString(string s) {
        int score = 0, n = s.length(), prev = (int) s[0];

        for (int i = 1; i < n; i++) {
            int current = (int) s[i];
            score += abs(prev-current);
            prev = current;
        }
        return score;
    }
};

File: Score of a String.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0
        n = len(s)
        prev = ord(s[0])
        for i in range(1, n):
            current = ord(s[i])
            score += abs(prev-current)
            prev = current
        return score
